### ISSUE TITLE
Check out Ranger Protocol (chain) integration

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#2789]
Feature branch first. Let's see if it works and if there's any gotchas.

**Comment by JamesSmartCell** [comment]
After a quick check it's mostly trivial to implement with one big gotcha.

Here's the standard node info for everything except one important detail:

https://doc.rangersprotocol.com/network.html
https://www.coincarp.com/chainlist/rangers-protocol-mainnet/

Resources:

https://rangersprotocol.com/img/logo.svg
https://cdn-images-1.medium.com/max/184/1*x3MI83ppZ8J_MUnrf4Kpjg@2x.png


The one missing detail is how to fetch existing transactions. It's unknown if the rangerscan supports API.

It should be easy to find it, as rangerscan looks like a fork of covalent. Compare it with PolygonScan:

RangerScan: https://scan.rangersprotocol.com/

PolygonScan: https://polygonscan.com/

I tried a few things but couldn't find any API to fetch transactions:

https://scan.rangersprotocol.com/api?action=txlist&address=0xc41bbf3810f754014441a70e4201be3e2972e5a7

I also tried Covalent directly, but it's not supported:

https://api.covalenthq.com/v1/2025/address/0xc41bbf3810f754014441a70e4201be3e2972e5a7/transactions_v2/?block-signed-at-asc=true&key=...

**Comment by hboon** [comment]
@JamesSmartCell thanks for checking, let's see if our friends can help. Checking…

**Comment by JamesSmartCell** [comment]
Yep good idea - the explorer is the same code as Polygonscan so it's a fork, just need to switch on the API

**Comment by hboon** [comment]
Hold for now.
